### ISSUE TITLE
Packaging Errors due to automatic debug symbols upload

### ISSUE THREAD

**Issue by ChristianBehrendt** [getsentry/sentry-unreal#163]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which version of the SDK?
0.1.1

How did you install the package?

Git

Which version of Unreal?

5.0

### Steps to Reproduce

1. Project Settings | Plugins | Sentry | Upload Debug Symbols Automatically - false

### Expected Result

Sentry won't try to automatically upload debug symbols.

### Actual Result

Sentry automatically tries to upload debug symbols. There is a "PostBuild-1.bat" that gets created and executed which tries to run a powershell-script to upload the symbols. 
But there is also a bug in the bat as it fails to run for projects that includes spaces in their name, since the pathing wont work then and the power shell script won't be found. 
In conclusion this leads to the packaging being aborted.

**Comment by tustanivsky** [comment]
@ChristianBehrendt The fix for this issue (#94) was already merged into the `main` branch and will be released with the next plugin version.

**Comment by tustanivsky** [comment]
Het @ChristianBehrendt, the fix is already live in the new release [0.2.0](https://github.com/getsentry/sentry-unreal/releases/tag/0.2.0). Closing it now, though feel free to reopen the issue if the problem persists
